### DISCUSSION TITLE
PnP Search Filter Check box template does not show search text box

### DISCUSSION THREAD

**Discussion by apolorc** [microsoft-search/pnp-modern-search#3425] [Q&A]
Hi guys,

I am using version 4.2.3 of PnP Modern Search and need to using filter with check box template with text box search  facility.

Each customize field filter are associated to a RefinableString, and no multi-value configuration, but only show a list of check box one for each values, and not any text box.

Can you please help me?

**Comment by wobba** [comment]
Using single value and combo template should work https://microsoft-search.github.io/pnp-modern-search/usage/search-filters/ as combo template is the one with filtering.

**Comment by kasperbolarsen** [comment]
Can you verify the data from the API using one of the tools suggested here https://microsoft-search.github.io/pnp-modern-search/using-query-tools-to-verify-issues/

**Comment by apolorc** [comment]
Hi guys,

Thanks for your reply, I just solved the issue. Update lastest release version v.4.10 and we get check box filter with text box.
Can you please closed this discussion?
Thanks in advanced

**Reply by wobba** [reply]
Seems we should update docs as well 🙂
